<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>702</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>1111</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mainLayout">
     <item>
      <widget class="QFrame" name="mainFrame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>10</y>
          <width>481</width>
          <height>491</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="paletteLayout">
         <item>
          <widget class="QFrame" name="paletteFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QTabWidget" name="resultTab_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>461</width>
              <height>451</height>
             </rect>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="jobTab">
             <attribute name="title">
              <string>Job</string>
             </attribute>
             <widget class="QTreeView" name="jobView">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>431</width>
                <height>331</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="removeJob">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>350</y>
                <width>121</width>
                <height>21</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
             <widget class="QPushButton" name="editJob">
              <property name="geometry">
               <rect>
                <x>320</x>
                <y>350</y>
                <width>121</width>
                <height>21</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
             <widget class="QPushButton" name="addJob">
              <property name="geometry">
               <rect>
                <x>160</x>
                <y>350</y>
                <width>131</width>
                <height>21</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
             <widget class="QPushButton" name="loadJob">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>390</y>
                <width>211</width>
                <height>21</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Load Job</string>
              </property>
             </widget>
             <widget class="QPushButton" name="saveJob">
              <property name="geometry">
               <rect>
                <x>230</x>
                <y>390</y>
                <width>211</width>
                <height>21</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Save Job</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="ioTab">
             <attribute name="title">
              <string>Tool</string>
             </attribute>
            </widget>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>0</y>
              <width>111</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Palette</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>601</width>
          <height>491</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QGraphicsView" name="cameraView">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>571</width>
           <height>391</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor" stdset="0">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="liveCamera">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Live Camera</string>
         </property>
        </widget>
        <widget class="QPushButton" name="triggerCamera">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Trigger Camera</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="focusBar">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>440</y>
           <width>571</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
        <widget class="QLCDNumber" name="executionTime">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>10</y>
           <width>64</width>
           <height>23</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Filled</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="runJob">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>10</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Run Job</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="jobFrame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>500</y>
          <width>601</width>
          <height>131</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QListView" name="toolView">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>421</width>
           <height>111</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="addTool">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>10</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Show</string>
    </property>
    <addaction name="actionList_tool"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
  </widget>
  <action name="actionList_tool">
   <property name="text">
    <string>SED Tool</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
